src/Workleap.DomainEventPropagation.Abstractions/DomainEventAttribute.cs

```diff
@@ -3,7 +3,11 @@
 [AttributeUsage(AttributeTargets.Class, Inherited = false)]
 public sealed class DomainEventAttribute : Attribute
 {
-    public DomainEventAttribute(string name)
+    public DomainEventAttribute(string name) : this(name, EventSchema.EventGridEvent)
+    {
+    }
+
+    public DomainEventAttribute(string name, EventSchema eventSchema = EventSchema.EventGridEvent)
```

Comment:
Considérant que l'autre constructeur donne déjà EventGrid comme default, lui devrait pas plutôt être du genre?
```suggestion
    public DomainEventAttribute(string name, EventSchema eventSchema)
```